File: STP/src/gestionAplicacion/Planeacion/Producto.java
package gestionAplicacion.Planeacion;

public class Producto {
	private String tipo;
	private Double peso;
	
	public Producto(String tipo, Double peso) {
		super();
		this.tipo = tipo;
		this.peso = peso;
	}
	public Producto() {
	}

	public String getTipo() {
		return tipo;
	}

	public void setTipo(String tipo) {
		this.tipo = tipo;
	}

	public Double getPeso() {
		return peso;
	}

	public void setPeso(Double peso) {
		this.peso = peso;
	}
}

File: STP-Python/gestionAplicacion/Operatividad/Producto.py
class Producto():

    def __init__(self, tipo, peso):
        self._tipo = tipo
        self._peso = peso

    def getTipo(self):
        return self._tipo

    def getPeso(self):
        return self._peso

    def setTipo(self, tipo):
        self._tipo = tipo

    def setPeso(self, peso):
        self._peso = peso
        